extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -804,4 +807,15 @@ export class AmpA4A extends AMP.BaseElement {
        metaData.cssUtf16CharOffsets[0],
        metaData.cssUtf16CharOffsets[1]);
  }
+
+  registerAlpHandler_(iframeWin) {
+    if (!isExperimentOn(this.win, 'alp-for-a4a')) {
+      return;
+    }
+    iframeWin.addEventListener('click', event => {
```

Comment:
Does this work? I think `iframeWin.document.documentElement` would be more idiomatic.